Bottom: c97e48b61744efc6bdd63bf987379c6babd49714
Top:    eef76ec7060aaee7337802328b1974f746aecc4f
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-05 12:53:53 +0100

hid/gtk: Turn on defines to debug a potential GTK-3.0 build


---

diff --git a/configure.ac b/configure.ac
index a802790..298c63f 100644
--- a/configure.ac
+++ b/configure.ac
@@ -759,6 +759,9 @@ $GTKGLEXT_PKG_ERRORS])]
 	GTKGLEXT_VER=`$PKG_CONFIG gtkglext-1.0 --modversion`
 	fi
 
+	GTK_CFLAGS="$GTK_CFLAGS -DGSEAL_ENABLE"
+	GTK_CFLAGS="$GTK_CFLAGS -DGDK_DISABLE_DEPRECATED -DGTK_DISABLE_DEPRECATED"
+
 	;;
 
       gcode|nelma|png )
